js/foundation.dropdown.js

```diff
@@ -312,7 +318,8 @@ class Dropdown {
       if(curPositionClass){
         this.$element.removeClass(curPositionClass);
       }
-      this.$element.addClass(this.options.positionClass)
+
+      this.$element.addClass(this.initialPositionClass)
```

Comment:
the code in `close()` and `_reposition()` is too much legacy and my fix not a good solution at all; please reject the PR.